File: Java/Array/Lab Test 01 Practice/Solve/StudentRankingSystem/src/Student.java
public class Student {

    private int id;
    private String name;
    private double gpa;

    public Student(int id, String name, double gpa) {
        this.name = name;
        this.id = id;
        this.gpa = gpa;
    }

    public int getId(){
        return this.id;
    }
    public String getName(){
        return this.name;
    }
    public double getGpa(){
        return this.gpa;
    }
    public String toString(){
        String result = "Student{id=" + this.id + ", name='"+ this.name + "'" + "', GPA=" + this.gpa +"}";
        return result;
    }
}

File: Python/Array/Lab 01 Practice/StudentRankingSystem/Student.py
class Student:
    def __init__(self, id, name, gpa):
        self.name = name
        self.id = int(id)
        self.gpa = float(gpa)

    def getName(self):
        return self.name

    def getId(self):
        return self.id

    def getGpa(self):
        return self.gpa

    def __str__(self):
        return f"Student{{id={self.id}, name='{self.name}', GPA={self.gpa}}}"